Cargo.toml

```diff
@@ -284,9 +284,9 @@ object_store = { version = "0.10", features = ["aws", "gcp", "azure", "http"] }
 once_cell = "1.18.0"
 p256 = { version = "0.13.2", features = ["ecdsa"] }
 parking_lot = "0.12.1"
-passkey-authenticator = { version = "0.2.0" }
-passkey-client = { version = "0.2.0" }
-passkey-types = { version = "0.2.0" }
+# TODO: Re-enable when passkey 0.4.0 is released
```

Comment:
Any indication when that would be?

Reply:
No clue

Reply:
Do you not think it would be preferable to temporarily allow the advisory? In case passkey decides to go another route and don't release these changes?

Reply:
Well, they released 0.4 today so let's see